ui-core/src/participant/landing/ConfiguredMedia.tsx

```diff
@@ -0,0 +1,88 @@
+import classNames from 'classnames'
+import React, { CSSProperties } from 'react'
+import { useApiContext } from '../../participant/ApiProvider'
+
+import { requireOptionalString, requirePlainObject, requireOptionalNumber }
+  from '../../participant/util/validationUtils'
+
+export type VideoConfig = {
+  videoLink: string
+  alt?: string
+  className?: string
+  style?: CSSProperties
+}
+
+export type ImageConfig = {
+  cleanFileName: string
+  version: number
+  alt?: string
+  className?: string
+  style?: CSSProperties
+}
+
+export type MediaConfig = ImageConfig | VideoConfig
+
+// TODO: Add JSDoc
```

Comment:
we made a change to our linting policy this summer -- we didn't use to require comments for const-declared functions.  Then when we decided we wanted to require those docs, we automatically added TODOs to the 300+ existing functions so that existing code would pass linting.  This is one of those (git is telling you this is a new file, but I just pasted it from the previous file).

Reply:
Normally I try to clean up the TODOs as I touch those old files, I just missed this one